docs/release/release-tasks.md

```diff
@@ -188,6 +188,9 @@ to a newer Go minor version according to our [backport policy](./../../CONTRIBUT
 1. Ensure CI is stable before cutting the release (e.g. by checking with the CI manager)
    Note: special attention should be given to image scan results, so we can avoid cutting a release with CVE or document known CVEs in release notes.
 2. Create and push the release tags to the GitHub repository:
+
+   **NOTE:** when cutting a release tag it will cause issues for users trying to download the latest release tag. See [issue 7889](https://github.com/kubernetes-sigs/cluster-api/issues/7889) for more details
```

Comment:
Can we add that this is resolved when the release is published? e.g. 

```suggestion
   **NOTE:** clusterctl will have issues installing providers between the time the release tag is cut and the Github release is published. See [issue 7889](https://github.com/kubernetes-sigs/cluster-api/issues/7889) for more details.
```

Reply:
updated.